diagnostics/ENSO_MSE/MSE/check_input_files.py

```diff
@@ -0,0 +1,53 @@
+

```

Comment:
check_input_files and check_input_files_OBS can be combined by making the filepath and list of variables arguments, and calling the routine like :
```
filepaths = ['path1', 'path']
varlist=['var1', 'var2','var3']
for f in filepaths:
    check_input_files(f, varlist) 
```